cirq-core/cirq/sim/clifford/stabilizer_state_ch_form.py

```diff
@@ -304,3 +292,101 @@ def reindex(self, axes: Sequence[int]) -> 'cirq.StabilizerStateChForm':
         copy.s = self.s[axes]
         copy.omega = self.omega
         return copy
+
+    def apply_x(self, g: common_gates.XPowGate, axis: int):
```

Comment:
Use `'cirq.XPowGate'` for type annotations.

*However*, also consider just passing in the exponent and phase rather than the whole gate: this feels like a low-level class so we should remove the `cirq.ops` dependency too.